documentation/docs/strategies/advanced-mm/price-source.md

```diff
@@ -1,9 +1,37 @@
 # External Pricing Source Configuration
 
+**Updated as of `v0.28.0`**
+
 By default, Hummingbot uses the market orderbook mid price (between the top bid and the top ask) as a starting price to calculate maker order prices. 
 With external pricing sources, you can now use external sources for the starting mid price such as an external **exchange** or a **custom API**.
 
-In a situation where the calculation of maker order prices from external sources would result in the order matching any existing orders on the order book, such order will be ignored. For example, if ETH-USDC market is currently displaying 109 bid and 111 ask. A specified external exchange is showing 99 bid and 101 ask on its book (mid price = 100). 2 maker orders will be proposed, a bid maker order at 98 (for 2% bid spread) and an ask maker order at 102 (for 2% ask spread). The 102 ask order will be ignored (as it would match the 109 bid order), only the bid order will be submitted to the exchange. 
+##How It Works
+
+In a situation where the calculation of maker order prices from external sources would result in the order matching any existing orders on the order book, such order will be ignored. 
+
+>**Example**: If `ETH-USDC` market is currently displaying 109 bid and 111 ask. A specified external exchange is showing 99 bid and 101 ask on its book (mid price = 100). 2 maker orders will be proposed, a bid maker order at 98 (for 2% bid spread) and an ask maker order at 102 (for 2% ask spread). The 102 ask order will be ignored (as it would match the 109 bid order), only the bid order will be submitted to the exchange. 
```

Comment:
This example might be clearer if you referenced a `ETH-USDT` market on the external exchange. This will help users understand how to use it.

Also, please explain why users should consider using this feature. The reason is that your current market may not have the best information about the real price. However, another market on a different exchange may have more information, either because it's larger or more liquid, or because it has information not yet known by the rest of the market.